index.html

```diff
@@ -15,43 +15,66 @@
-
-  <img src="img/blast.jpg" alt="Rocket blasting off">
-
-  <img src="img/sky.jpg" alt="Space sky image">
-
-  <img src="img/launch.jpg" alt="Launching rocket">
-
-  Can We Visit Planets?
-
-  Never in all their history have men been able truly to conceive of the world as one: a single sphere, a globe, having the qualities of a globe, a round earth in which all the directions eventually meet.
-
-  which there is no center because every point, or none, is center — an equal earth which all men occupy as equals. The airman's earth, if free men make it.
-
-  Let's Go For A Walk
-
-  The Earth was small, light blue, and so touchingly alone, our home that must be defended like a holy relic. The Earth was absolutely round.
-
-  It suddenly struck me that that tiny pea, pretty and blue, was the Earth. I put up my thumb and shut one eye, and my thumb blotted out the planet Earth.
-
-  Sign Up
-
-  <img src="img/space-walk.jpg" alt="Space walking astronaut">
-
-  Copyright SpaceWalkers 2018
+  <!-- HEADER CONTAINER -->
+  <div class = 'header-container'>
+              <h1>SpaceWalkers</h1>
+              <nav class="nav">
+                <a class="nav-link" href="#">Home</a>
+                <a class="nav-link" href="#">About Us</a>
+                <a class="nav-link" href="#">Blog</a>
+                <a class="nav-link" href="#">Contact</a>
+                <a class="nav-link" href="#">Sign In</a>
+              </nav>
+  </div><!-- header-container -->
+
+  <!-- MAIN CONTAINER -->
+  <div class = 'main-container'>
+      <section class = 'Lift-off'></section>
+        <img class = 'space-jumbo' src="img/main-header.jpg" alt="Space view of Earth">
+        <h2>Lift Off</h2>
+        <p>As we got further and further away, it [the Earth] diminished in size. 
+          Finally it shrank to the size of a marble, the most beautiful you can imagine. 
+          That beautiful, warm, living object looked so fragile, so delicate, 
+          that if you touched it with a finger it would fall apart. Seeing this has to change a man.
+        </p>
+      </section><!-- lift-off -->
+      <section class = 'Can-We-Visit'>
+          <img src="img/blast.jpg" alt="Rocket blasting off">
+          <img src="img/sky.jpg" alt="Space sky image">
+          <img src="img/launch.jpg" alt="Launching rocket">
+          <h2>Can We Visit Planets?</h2>
+          <p>Never in all their history have men been able truly to conceive of the world as one: 
+            a single sphere, a globe, having the qualities of a globe, 
+            a round earth in which all the directions eventually meet.
+          </p>
+          <p>which there is no center because every point, or none, is center — 
```

Comment:
Your html is very clean. I would just recommend following the same indenting/styling throughout your html file